www/locations/monterrey/index.html

```diff
@@ -0,0 +1,14 @@
+{% extends "site.html" %}

```

Comment:
Please remove this file. Only external websites are permitted now on pyladies.com to link to your website. You can see how to create in the Please complete the directions here: https://github.com/pyladies/pyladies/blob/main/README.md#registering-your-pyladies-chapter-and-onbtaining-a-website.